contracts/SpokePool.sol

```diff
@@ -60,8 +60,8 @@ abstract contract SpokePool is
     WETH9Interface public wrappedNativeToken;
 
     // Any deposit quote times greater than or less than this value to the current contract time is blocked. Forces
-    // caller to use an approximately "current" realized fee. Defaults to 1 hour.
-    uint32 public depositQuoteTimeBuffer;
+    // caller to use an approximately "current" realized fee.
+    uint32 public constant depositQuoteTimeBuffer = 3600;
```

Comment:
TODO: Figure out if this is upgrade-safe

Reply:
If not, we'll need to define a new constant DEPOSIT_QUOTE_TIME_BUFFER at the bottom of the contract's variables

Reply:
https://github.com/across-protocol/contracts-v2/pull/371#discussion_r1406843568

I don't think we ever actually change `depositQuoteTimeBuffer` in the tests and just leave it hardcoded to 3600 as set in the constructor. So IMO there's no functional difference making this constant or immutable since we can just as easily change it going forwards

Reply:
Nevermind, as discussed offline, `immutable` for some variables like `depositQuoteTimeBuffer` allows us to set a different value for diff. child contracts which could come in handy given the block time disparity between chains